### PR TITLE
audit log dashboard documentation

### PR DESCRIPTION
**Pull request by clutroth**

### CODE REVIEW

File: kibana.md [hunk 1]
```diff
@@ -399,6 +399,28 @@ The result may look something like this:
 http://localhost:5601/login?nextUrl=%2Fapp%2Fkibana%23%2Fvisualize%2Fedit%2F28dcde30-2258-11e8-82a3-af58d04b3c02%3F_g%3D%28%29&jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWV9.TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ
 ```
 
+## Audit log
+Audit log feature is wide described in [📖docs for Elasticsearch plugin](elasticsearch.md#audit-logs).
```
Comment (sscarduzio) [inline kibana.md]: > **The** audit log feature is **widely** described in ...

Remember to use the articles (please add them also to the rest of the PR) :)

File: kibana.md [hunk 2]
```diff
@@ -399,6 +399,28 @@ The result may look something like this:
...
+Visualization loading button is located in _Audit_ tab. 
+![audit log tab](audit_tab.png)
+
+Click _Load_ button to load visualization.
+_Override_ checkbox allows you to load current version saved objects. 
+It will override your's current settings. 
+
+![loading visualization](load_audit_dashboard.png)
+
+Currently loading dashboard will fetch tou your's kibana
```
Comment (sscarduzio) [inline kibana.md]: I don't understand

File: kibana.md [hunk 3]
```diff
@@ -399,6 +399,28 @@ The result may look something like this:
...
+* index pattern for `readonlyrest_audit-*`
+* dashboard `ReadonlyREST Audit Log`
+* visualizations
+
+### Dashboard
+Dashboard, by default has few basic visualizations. 
+They're covers security, accessibility, performance, and performance diagrams.
```
Comment (sscarduzio) [inline kibana.md]: > They cover security, access logs, and performance metrics.

File: kibana.md (hunk 2 above)
Comment (clutroth) [reply kibana.md]: I rewritten it. I hope, I expressed myself better.

File: kibana.md (hunk 1 above)
Comment (clutroth) [reply kibana.md]: Did you mean to use `# Audit log`, '## Loading visualization', and so on?

**Pull request merged**